test/unit/range/views/view_minimiser_hash_test.cpp

```diff
@@ -0,0 +1,102 @@
+// -----------------------------------------------------------------------------------------------------
```

Comment:
For similar tests there were some reviews in other PRs, right?
Can you incorporate the changes here as well, if any?

Reply:
Yeah, the iterator test has to be added, I will do so once the iterator test is incorporated in kmer_hash and minimiser_view, otherwise even the travis tests will fail for this PR.

Reply:
test template is still missing :)